cltk/tests/test_nlp/test_lemmatize.py

```diff
@@ -102,27 +105,71 @@ def test_regex_lemmatizer(self):
         lemmas = lemmatizer.lemmatize(tokens)
         self.assertEqual(lemmas, target)
 
-    def test_latin_pp_lemmatizer(self):
-        """Test latin_pp_lemmatizer()"""
-        pattern = [(r'(\w*)[a|ie]bimus\b', 1)]
-        pps = { 'amo': [1, 'am', 'amare', 'amau', 'amat'] }
-        lemmatizer = PPLemmatizer(pattern, pps=pps)
-        test_str = 'amabimus'
-        target = [('amabimus', 'amo')]
-        jv_replacer = JVReplacer()
-        tokenizer = WordTokenizer('latin')
-        test_str = test_str.lower()
-        test_str = jv_replacer.replace(test_str)
-        tokens = tokenizer.tokenize(test_str)
-        lemmas = lemmatizer.lemmatize(tokens)
-        self.assertEqual(lemmas, target)
+# Removed temporarily
+    # def test_latin_pp_lemmatizer(self):
+    #     """Test latin_pp_lemmatizer()"""
+    #     pattern = [(r'(\w*)[a|ie]bimus\b', 1)]
+    #     pps = { 'amo': [1, 'am', 'amare', 'amau', 'amat'] }
+    #     lemmatizer = PPLemmatizer(pattern, pps=pps)
+    #     test_str = 'amabimus'
+    #     target = [('amabimus', 'amo')]
+    #     jv_replacer = JVReplacer()
+    #     tokenizer = WordTokenizer('latin')
+    #     test_str = test_str.lower()
+    #     test_str = jv_replacer.replace(test_str)
+    #     tokens = tokenizer.tokenize(test_str)
+    #     lemmas = lemmatizer.lemmatize(tokens)
+    #     self.assertEqual(lemmas, target)
+
+# Removed temporarily
+    # def test_roman_numeral_lemmatizer(self):
+    #     """Test roman_numeral_lemmatizer()"""
+    #     rn_patterns = [(r'(?=^[MDCLXVUI]+$)(?=^M{0,4}(CM|CD|D?C{0,3})(XC|XL|L?X{0,3})(IX|IV|IU|V?I{0,3}|U?I{0,3})$)', 'NUM'), (r'(?=^[mdclxvui]+$)(?=^m{0,4}(cm|cd|d?c{0,3})(xc|xl|l?x{0,3})(ix|iv|iu|v?i{0,3}|u?i{0,3})$)', 'NUM')]
```

Comment:
What's the plan for re-insertion of the roman_numeral_lemmatizer?

Reply:
That one is actually less experimental than the others—can just put it back now.